selfswab/tasks.py

```diff
@@ -82,6 +84,12 @@ def poll_meditech_api_for_results():
                     if result.get("verifyDateTime"):
                         registration.authorized_timestamp = result.get("verifyDateTime")
 
+                    if not registration.pdf_media_id and result.get("pdf_path"):
+                        content = requests.get(result["pdf_path"]).content
+                        registration.pdf_media_id = upload_turn_media(content)
+                    else:
```

Comment:
Won't this also trigger if `registration.pdf_media_id` has a value? I think this should rather be `elif not registration.pdf_media_id and not result.get("pdf_path")`

Reply:
let me create a test for this, thanks